Lecture_9/src/utils.js

```diff
@@ -0,0 +1,9 @@
+export function mergeObjects (...objects) {
+  return Object.assign({}, ...objects)
+}
+
+export async function getUserInfo (id) {
+  const user = await fetch(`https://jsonplaceholder.typicode.com/users/${id}`)
+  const userJSON = await user.json()
+  return userJSON
```

Comment:
Not a mistake, just a different possible approach. If you aren't planning any additional operations with `userJSON` variable before returning it, you can avoid the declaration and simply return `user.json()`.
```suggestion
  return user.json()
```